xwiki-platform-core/xwiki-platform-legacy/xwiki-platform-legacy-events-hibernate/xwiki-platform-legacy-events-hibernate-api/src/main/java/org/xwiki/eventstream/script/EventStreamScriptService.java

```diff
@@ -56,6 +56,7 @@
 @Component
 @Singleton
 @Named("eventstream")
+@Deprecated
 public class EventStreamScriptService implements ScriptService
```

Comment:
it cannot extends the new `EventScriptService` since you implemented there some APIs of it?

Reply:
It's not very clear what you mean. Are you talking about keeping "eventstream" for EventScriptService ?

Reply:
No I meant to write `public class EventStreamScriptService extends EventScriptService`, it looks like the code of `getRecordableEventDescriptors` and `getDescriptorForEventType` is now duplicated no?

Reply:
This made me think of a less disturbing change: I reverted the changed on standard side and just moved the legacy related method to override of the script service on legacy side. That way the script service ID does not change.